hazelcast/src/main/java/com/hazelcast/internal/management/ManagementCenterService.java

```diff
@@ -164,7 +157,7 @@ static String cleanupUrl(String url) {
         return url.endsWith("/") ? url : url + '/';
     }
 
-    public void start() {
+    private void start() {
```

Comment:
Yes, you are right. fixed https://github.com/hazelcast/hazelcast/pull/9571/commits/f75fc2f44a0778c910e3f1f9e0f404368446cde3#diff-3850ee7b4bd9df3672e6c4b18d1ec70cR131